multimedia/vapoursynth/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            vapoursynth vapoursynth 51 R
+
+categories              multimedia python
+platforms               darwin
+license                 LGPGL-2.1
```

Comment:
Typo: That should be `LGPL`. Looking at the source files like src/core/vscore.cpp, the license header actually reads:
```
* VapourSynth is free software; you can redistribute it and/or
* modify it under the terms of the GNU Lesser General Public
* License as published by the Free Software Foundation; either
* version 2.1 of the License, or (at your option) any later version.
```
So the license field should be `LGPL-2.1+`.

Reply:
Thanks for catching that, fixed.